.circleci/config.yml

```diff
@@ -43,6 +44,18 @@ jobs:
           command: |
             echo "export TAG=ci-build-$CIRCLE_BRANCH" >> $BASH_ENV
       - *build_and_push
+  build_1804_deb:
+    <<: *container_config
+    steps:
+      - checkout
+      - *setup_remote_docker
+      - *install_docker_client
+      - run:
+          name: Configure docker image tag
+          command: |
+            echo "export TAG=1804_deb_pkgs" >> $BASH_ENV
```

Comment:
Not sure why the `deb_pkgs` suffix, this should be general builder package based on ubuntu 18.04. I'd expect `ubuntu-18` or similar

Reply:
Same as https://github.com/aeternity/docker-builder/pull/13#discussion_r261497741.

Reply:
Fixed in d30a4bc